azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -1074,6 +1077,11 @@ private boolean isServiceRequired() {
    * @throws ExecutorManagerException
    */
   private void deletePod(final int executionId) throws ExecutorManagerException {
+    final ExecutableFlow flow = this.executorLoader.fetchExecutableFlow(executionId);
+    Status s = flow.getStatus();
```

Comment:
Nit: maybe use more descriptive name like `flowStatus` or even `status`.

Reply:
changed to status, ty.